src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/memory.hpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "ov_models/utils/ov_helpers.hpp"
+
+namespace ov {
+namespace test {
+
+using MemoryLayerTestParams = std::tuple<
+        ngraph::helpers::MemoryTransformation,   // Apply Memory transformation
```

Comment:
need to port `ngraph::helpers::MemoryTransformation`, may be done in scope of this PR

Reply:
done